From 13262a18f804638b40213a865e0a72e33123ccf1 Mon Sep 17 00:00:00 2001
From: Sumit Bose <sbose@redhat.com>
Date: Tue, 14 Oct 2014 16:52:04 +0200
Subject: [PATCH 3/4] ipa_subdomains_handler_master_done: initialize
 reply_count

This patch should mainly silence a false-positive Coverity warning but
since further processing depends on this variable I think it is a good
idea anyways.

Reviewed-by: Pavel Reichl <preichl@redhat.com>
---
 src/providers/ipa/ipa_subdomains.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/providers/ipa/ipa_subdomains.c b/src/providers/ipa/ipa_subdomains.c
index eb172fdfc05ac4e482174f01d89ad28db1498fc1..c61c1c666908ec23f8a92e5568222e55ec47be0a 100644
--- a/src/providers/ipa/ipa_subdomains.c
+++ b/src/providers/ipa/ipa_subdomains.c
@@ -1276,7 +1276,7 @@ static void ipa_subdomains_handler_master_done(struct tevent_req *req)
 {
     errno_t ret;
     int dp_error = DP_ERR_FATAL;
-    size_t reply_count;
+    size_t reply_count = 0;
     struct sysdb_attrs **reply = NULL;
     struct ipa_subdomains_req_ctx *ctx;
 
-- 
1.9.3

